package.json

```diff
@@ -8,7 +8,7 @@
   "scripts": {
     "crawl": "node ./cli/crawl-cli",
     "test": "npm run lint && tsc && npm run unit",
-    "unit": "find . -name \"*.test.js\" -not -path \"./node_modules/*\" -exec node {} \\;",
+    "unit": "node ./tests/runTests.js",
```

Comment:
Some test failures didn't cause the whole testing to fail, so I had to write my own small runner to work around this. I guess, at some point, I should consider just using one of existing test runners.